### ISSUE TITLE
Accessibility: Not enough contrast

### ISSUE THREAD

**Issue by JoeProgrammer88** [SpeakingInBits/PC2#204]
From accessibility checker:
Elements that have texts should meet a minimum contrast ratio of 4.5:1 between their foreground (usually text color) and background color.

Failed elements on home page:
```html
<a class="name" href="/" >Pierce County Coalition for <br > Developmental Disabilities (PC2)</a>
<div id="menu-title">MENU</div>
<p> PC2's mission: <br > <em> "Empowering individuals with developmental disabilities and their families." </em> </p>
<em> "Empowering individuals with developmental disabilities and their families." </em>
<p> PC2's vision: <br > <em> "All people with developmental disabilities have access<br > to the services and programs that meet their needs." </em> </p>
<em> "All people with developmental disabilities have access<br > to the services and programs that meet their needs." </em>
<li>Provides resource links and follow-up/follow-along services for families.</li>
<li>Publishes a quarterly newsletter with relevant articles and information.</li>
<li>Creates and distributes a comprehensive Resource Guide for Pierce County.</li>
<li> Conducts educational meetings on a variety of topics from birth to three services, housing/residential issues, transition from high school to adult services, employment, legal overviews, and more. ...
```

Successful elements
```html
<p class="text-white">253.564.0707</p>
<p class="text-white">877-DD-LINKS</p>
<p>3716 Pacific Ave, STE A</p>
<p>Tacoma, WA 98418</p>
<p>© 2023 by PC2 for DD</p>
<a class="nav-link text-dark" href="/Identity/Account/Login" >Login</a>
```
